sdk/core/azure-core-tracing-opentelemetry/azure/core/tracing/ext/opentelemetry_span/__init__.py

```diff
@@ -38,7 +38,7 @@
 __version__ = VERSION
 
 
-def _get_headers_from_http_request_headers(headers: "Mapping[str, Any]", key: str):
+def _get_headers_from_http_request_headers(headers: "MutableMapping[str, Any]", key: str):
```

Comment:
Why Mutable? We're just reading it here, and "MutableMapping" is a subtype of "Mapping", so Mapping should not matter

Reply:
sounds fair - Changing this to Mapping - keeping the set method to mutable